tests/Doctrine/Tests/Persistence/ObjectManagerDecoratorTest.php

```diff
@@ -22,7 +22,7 @@ public function __construct(ObjectManager $wrapped)
 
 class ObjectManagerDecoratorTest extends TestCase
 {
-    /** @var MockObject|ObjectManager */
+    /** @var MockObject&ObjectManager */
```

Comment:
About this, it was included in PHPStorm apparently in 2018:https://blog.jetbrains.com/phpstorm/2018/09/phpstorm-2018-3-eap-183-2635-12/

Let me now if we should use `@psalm-var`.

Reply:
PHPStorm indeed added parsing for it in 2.018 (it was still _treating_ it as a union type internally as it did not support intersection types, but that's not a reason to avoid it in favor of writing a union type directly).

So +1 from me